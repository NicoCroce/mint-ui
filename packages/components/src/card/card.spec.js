import * as React from "react";
import { cleanup, render} from "@testing-library/react";
import Card from "./index";

describe("Card", () => {
    test("renders correctly", () => {
        const { asFragment } = render(
            <Card title="Default size card" extra={<a href="#">More</a>} style={{ width: 300 }}>
                <p>Card content</p>
                <p>Card content</p>
                <p>Card content</p>
            </Card>
        );
        expect(asFragment()).toMatchSnapshot();
    });
});
