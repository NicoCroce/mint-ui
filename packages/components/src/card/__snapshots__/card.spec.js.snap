// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<DocumentFragment>
  <div
    class="card__Wrapper-sc-1au9sup-0 iufszp"
    style="width: 300px;"
  >
    <div
      class="card__Header-sc-1au9sup-1 kziSce"
    >
      <div
        class="card__HeadWrapper-sc-1au9sup-2 fYAPjq"
      >
        <div
          class="card__Title-sc-1au9sup-3 hyrFKT"
        >
          Default size card
        </div>
        <div
          class="card__Extra-sc-1au9sup-4 szDyR"
        >
          <a
            href="#"
          >
            More
          </a>
        </div>
      </div>
    </div>
    <div
      class="card__Body-sc-1au9sup-5 fueqQ"
    >
      <p>
        Card content
      </p>
      <p>
        Card content
      </p>
      <p>
        Card content
      </p>
    </div>
  </div>
</DocumentFragment>
`;
